saos-enrichment/src/main/java/pl/edu/icm/saos/enrichment/upload/EnrichmentTagUploadController.java

```diff
@@ -123,7 +120,7 @@ private void checkContentType(String contentType) {
     
     @ExceptionHandler(ServiceException.class)
     private ResponseEntity<ServiceResponse> serviceException(ServiceException e) {
-        return errorResponse(e.getMainMessage(), e, BAD_REQUEST);
+        return errorResponse(e.getMainMessage(), e, enrichmentTagUploadMessageHttpStatusMapper.getHttpStatus(e.getMainMessage()));
     }
 
     @ExceptionHandler(Exception.class)
```

Comment:
in method `errorResponse` below you have `log.warn("", e);` which will produce empty log message